ckanext/unaids/auth.py

```diff
@@ -1,12 +1,33 @@
 # coding: utf
-import ckan.logic as logic
 import ckan.model as model
 import ckan.plugins.toolkit as toolkit
 
 
+def dataset_lock(context, data_dict):
+    # Sysadmins only
+    return {
+        'success': False,
+        'msg': 'You cannot carry out this action'
+    }
+
+
+@toolkit.chained_auth_function
+@toolkit.auth_sysadmins_check
+def package_update(next_auth_action, context, data_dict):
+    result = next_auth_action(context, data_dict)
+    locked = context['package'].extras.get("locked", 'false')
+    if locked in ['true', True]:
```

Comment:
!!!!! Try this:
```
>>> locked = 'True'
>>> locked in ['true', True]
False
```
Better to properly convert value to boolean on variable def:
```
locked = toolkit.asbool(context['package'].extras.get("locked", 'false'))
```

Reply:
Yes I wasn't happy with this either. `toolkit.asbool` is what I needed. Thankyou!